www/kanban/inner.js

```diff
@@ -1150,8 +1150,14 @@ define([
                 allTags.forEach(function (t) {
                     var tag;
                     $list.append(tag = h('span', {
-                        'data-tag': t
+                        'data-tag': t,
+                        'tabindex': 0
                     }, t));
+                    $(tag).keydown(function (e) {
```

Comment:
Small optimization possible here. You can use the `$tag` variable defined just after to avoid initializing a jQuery object twice:
```javascript
var $tag = $(tag).click(function () {
    // ...
}).keydown(function (e) {
    // Use $tag here instead of $(tag)
});